assessments/0002-notary-project.md

```diff
@@ -0,0 +1,176 @@
+| “New contributor” getting started content | ✅  |     |     |     |     |
+| Project governance documentation          | ✅  |     |     |     |     |
+
+### Comments
+
+_Provide comments for each rating above, 1-2 sentences max, bullet point list_
+
+**Communication methods documented**
+- There is a Contributing & Conversations section in the main README (tho, it is mis-linked in the table of contents)
+
+**Beginner friendly issue backlog**
+- There are "Documentation," "spec," and "good first issue" issue tags
+- Most issues seem to link out to hackmd.io notes, but those note are are well written and informative
+
+**“New contributor” getting started content**
+- Same as above, there is a Contributing & Conversations section, but there are not yet any instructions for contributions.
+
+**Project governance documentation**
+- This appears to be actively being worked on, and there is a "Governance" issue tag in use. 
+
+
+### Recommendations
+
+Most of the issues in this section are a function of not having a website yet, and the project still being in a building phase.
+- As we go through the site build project, we should put together a skeleton IA that can be filled in as the project grows.
+
+Regarding beginner friendliness, as we move through the project, we should consider processes/documents for:
+* Triaging docs issues
+* Clearly marking a way for new contributors to make code or documentation contributions (i.e. a “good first issue” label), and define what makes a good first issue
+* Maintaining issues for staleness
+* New contributors/your first contribution
+* Letting new users know where to get help
+
+
+## Website
+
+Criteria
+- 1 = (Is not present or requires significant work)
+- 3 = (is present, but needs work) 
+- 5 = (is executed extremely well or no improvement required) |
+
+| Criteria                  | 1   | 2   | 3   | 4   | 5   |
+| ---                       | --- | --- | --- | --- | --- |
+| Branding and design       |     | ✅  |     |     |     |
+| Case studies/social proof | ✅  |     |     |     |     |
+| Maintenance planning      | ✅  |     |     |     |     |
+
+### Comments
+
+Currently there is no website, so some of this is not applicable (as we're starting the project of building one)
+
+**Branding and design**:
+- Artwork exists https://github.com/cncf/artwork/tree/master/projects/notary
+ 
+**Case studies/social proof**:
+- I'm not sure this is applicable for V2 yet. V1 may have some of this information.
+
+**Maintenance planning**: 
+- N/A
+
+### Recommendations
+
+There may not be much now, but as we collect information for this site, we can ask the following questions:
+
+**Regarding Case studies/social proof**:
+* Are there case studies available for the project?
+* Are there user testimonials available?
+* Can we start a blog? 
+* Are there community talks for the project 
+* Is there a logo wall of users/participating organizations?
+
+
+## Final Recommendations
```

Comment:
For this section, please see [what I did for project Contour](https://github.com/cncf/techdocs/blob/main/assessments/0001-contour.md#final-recommendations).

Please also refer to the [instructions in the template](https://github.com/cncf/techdocs/blob/main/assessments/template.md#recommendations):

> From the recommendations above, lis the top 1-3 concerns for this particular project and expand on them in enough detail that you could either: - Pass the work off to a contractor or other member of the CNCF techdocs team - Write a GitHub issue for the project team and place it in the backlog and someone not involved in the docs assessment process could execute on it

_Each issue in this section_ should be, for lack of a better word, a fully formed GitHub issue or even an umbrella issue. 

I'm not going to review what you've written here in detail, because I think this section needs to be rewritten.